<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYIntegrationCurveDockGeneralTab</class>
 <widget class="QWidget" name="XYIntegrationCurveDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>858</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="2" colspan="4">
    <widget class="QComboBox" name="cbDataSourceType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="4">
    <widget class="ResizableTextEdit" name="teComment"/>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lDataSourceType">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lDataSourceCurve">
     <property name="text">
      <string>Curve:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lXColumn">
     <property name="text">
      <string>x-Data:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lYColumn">
     <property name="text">
      <string>y-Data:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lXRange">
     <property name="text">
      <string>x-Range:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QCheckBox" name="cbAutoRange">
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="4">
    <widget class="QFrame" name="frameRange">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lMaxDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lMinDateTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Minimum:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditMin">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditMax">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leMin"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leMax"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="4">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lIntegration">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Integration</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lMethod">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="5">
    <widget class="QComboBox" name="cbMethod"/>
   </item>
   <item row="13" column="1" colspan="3">
    <widget class="QCheckBox" name="cbAbsolute">
     <property name="text">
      <string>Absolute area</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lEval">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lResults">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Results:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="5">
    <widget class="QTextEdit" name="teResult">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0" rowspan="2" colspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>266</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="0">
    <widget class="QCheckBox" name="chkVisible">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item row="18" column="4">
    <widget class="QPushButton" name="pbRecalculate">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="lPlotRange">
     <property name="text">
      <string>Plot ranges:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="2" colspan="2">
    <widget class="QComboBox" name="cbPlotRanges"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
